stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -488,28 +486,48 @@ public struct ${Self}<Pointee>
   }
 
 %  if mutable:
+  @available(swift, obsoleted: 5.0.0, renamed: "initialize(repeating:count:)")
+  public func initialize(to newValue: Pointee, count: Int = 1) { 
+    initialize(repeating: newValue, count: count)
+  }
+
   /// Initializes this pointer's memory with the specified number of
   /// consecutive copies of the given value.
   ///
   /// The destination memory must be uninitialized or the pointer's `Pointee`
-  /// must be a trivial type. After a call to `initialize(to:count:)`, the
+  /// must be a trivial type. After a call to `initialize(repeating:count:)`, the
   /// memory referenced by this pointer is initialized.
   ///
   /// - Parameters:
-  ///   - newValue: The instance to initialize this pointer's memory with.
+  ///   - repeatedValue: The instance to initialize this pointer's memory with.
   ///   - count: The number of consecutive copies of `newValue` to initialize.
-  ///     `count` must not be negative. The default is `1`.
+  ///     `count` must not be negative. 
   @_inlineable
-  public func initialize(to newValue: Pointee, count: Int = 1) {
+  public func initialize(repeating repeatedValue: Pointee, count: Int) {
     // FIXME: add tests (since the `count` has been added)
     _debugPrecondition(count >= 0,
       "${Self}.initialize(to:): negative count")
```

Comment:
This _debugPrecondition message references the old func name

Reply:
fixed